vowpalwabbit/fb_parser/src/parse_example_flatbuffer.cc

```diff
@@ -42,9 +43,83 @@ int flatbuffer_to_examples(VW::workspace* all, io_buf& buf, VW::multi_ex& exampl
   return static_cast<int>(status.get_error_code() == VW::experimental::error_code::success);
 }
 
+bool read_span_flatbuffer(
+    VW::workspace* all, const uint8_t* span, size_t length, example_factory_t example_factory, VW::multi_ex& examples)
+{
+  // we expect context to contain a size_prefixed flatbuffer (technically a binary string)
+  // which means:
+  //
+  //    /   4b   /    *length b                           /
+  //    +--------+----------------------------------------+
+  //    | length | flatbuffer                             |
+  //    +--------+----------------------------------------+
+  //    |                   context                       |
+  //    +--------+----------------------------------------+
+  //
+  //    thus context.size() = sizeof(length) + length
+  io_buf unused;
+
+  // TODO: How do we report errors out of here? (This is a general API problem with the parsers)
```

Comment:
Yeah.  This is not great.  Does this introduce exceptions to clientlib during parsing?

Reply:
Regardless given that we should get this feature out, probably ok to live with this and improve later if needed.